new/rfcXXX-distro-sugar-helpers.md

```diff
@@ -0,0 +1,159 @@
+---
+RFC: unassigned
+Title: My Title
+Author: Alan Smithee <asmithee@example.com>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Title
+
+Merging chef-sugar platform and `platform_family` helpers, along with higher level helpers.
```

Comment:
and that's not just me, we shot that syntax down in this discussion:  https://github.com/chef/chef-rfc/pull/77

plus there's this work with largely replaces it (that i still need to finish very soon):  https://github.com/chef/chef-rfc/pull/238